docs/src/tutorial/pure.rst

```diff
@@ -168,13 +168,9 @@ Static typing
 
     cython.declare(x=cython.int, y=cython.double)  # cdef int x; cdef double y
 
-  It can also be used to type class constructors::
+  It can also be used to type class constructors:
 
-    class A:
-        cython.declare(a=cython.int, b=cython.int)
```

Comment:
Hmm, this example makes no sense. Cython will ignore this declaration just like Python does. Python classes do not have typed attributes.

I don't know what it was meant to show - maybe an extension type (that lacks an `@cclass` decorator)? But that is already shown in the example right below...

I think this example can just be deleted and the extension type example below extended to actually do something.